tests/cdc-endpos-between-transaction/copydb.sh

```diff
@@ -105,5 +102,8 @@ pgcopydb follow --resume --trace
 sql="select count(*) from category"
 test 26 -eq `psql -AtqX -d ${PGCOPYDB_SOURCE_PGURI} -c "${sql}"`
 
+kill -TERM ${COPROC_PID}
+wait ${COPROC_PID}
+
```

Comment:
Now the snapshot is needed until follow finishes!, is it okay? or better to go with --not-consistent snapshots?

Reply:
It's okay! We need a variety of tests covering multiple usages, I don't have a strong opinion on whether a particular test should hold on to its snapshot for what period of time. Tests should also be good implementation references when possible, so using consistency here is good too.